kernel/src/main/java/org/kframework/compile/GenerateSortPredicateRules.java

```diff
@@ -1,61 +1,69 @@
 // Copyright (c) K Team. All Rights Reserved.
 package org.kframework.compile;
 
-import org.apache.commons.lang3.mutable.MutableBoolean;
+import static org.kframework.Collections.*;
+import static org.kframework.definition.Constructors.*;
+import static org.kframework.kore.KORE.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
 import org.kframework.Collections;
 import org.kframework.attributes.Att;
 import org.kframework.builtin.BooleanUtils;
 import org.kframework.builtin.Sorts;
 import org.kframework.definition.Module;
-import org.kframework.definition.NonTerminal;
-import org.kframework.definition.Production;
 import org.kframework.definition.Rule;
 import org.kframework.definition.Sentence;
-import org.kframework.kore.K;
 import org.kframework.kore.KApply;
 import org.kframework.kore.KLabel;
 import org.kframework.kore.KRewrite;
 import org.kframework.kore.KVariable;
 import org.kframework.kore.Sort;
 import scala.collection.Set;
 
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Optional;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
-import static org.kframework.Collections.*;
-import static org.kframework.definition.Constructors.*;
-import static org.kframework.kore.KORE.*;
-
 /**
- * Generates sort predicates from the subsort hierarchy of the module. This module assumes that the backend implements
- * the following rules:
- * isSort(#token(Sort, _)) => true
- * isK(K) => true
- * isKItem(K1(K2)) => true
- * isKItem(#token(_, _)) => true
+ * Generates sort predicates from the subsort hierarchy of the module. This module assumes that the
+ * backend implements the following rules: isSort(#token(Sort, _)) => true isK(K) => true
+ * isKItem(K1(K2)) => true isKItem(#token(_, _)) => true
```

Comment:
This looks weird, and it might matter in this case.

Reply:
Yep, I agree that looks weird. I suspect that for comments it's possible to write them in such a way as to avoid this; will investigate